### DISCUSSION TITLE
umi4开启mpa模式不会运行app.ts么

### DISCUSSION THREAD

**Discussion by shen-kk** [umijs/umi#11171] [Q&A]
那么requestConfig怎么配置，Cannot read properties of null (reading 'applyPlugins')，在app.ts中写了配置，axios会报错读取不到conifg

**Comment by fz6m** [comment]
mpa 是单纯的多页应用，不能使用 umi spa 的插件等功能，只是单纯的构建多页面，一般情况无需使用 mpa 。
